openstack/clientconfig/results.go

```diff
@@ -17,9 +17,25 @@ type Cloud struct {
 	// API Version overrides.
 	IdentityAPIVersion string `yaml:"identity_api_version"`
 	VolumeAPIVersion   string `yaml:"volume_api_version"`
+
+	// Verify whether or not SSL API requests should be verified.
+	// Note! Type is not set to bool to be able to set correct default value.
+	Verify interface{} `yaml:"verify"`
```

Comment:
We can change this to `*bool` and the YAML package will account for it accordingly.

This way we can check for nil, true, and false.